src/phpDebug.ts

```diff
@@ -197,8 +197,9 @@ class PhpDebugSession extends vscode.DebugSession {
 
     /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
     protected convertDebuggerPathToClient(fileUri: string): string {
-        // convert the file URI to a path
-        const serverPath = decodeURI(url.parse(fileUri).pathname.substr(1));
+        // convert the file URI to a path. Don't remove starting slash on unix platforms.
+        let n:number = (process.platform === 'win32') ? 1 : 0;
```

Comment:
I though about this change, and I think it is not quite right. we are inspecting the OS platform of the _client_ and decide to modify the _server_ path. Isnt this dependent on the server OS rather than the client OS?